package.json

```diff
@@ -11,8 +11,12 @@
     "outputName": "js-test-results.xml"
   },
   "scripts": {
-    "start": "cd packages/rn-tester && npm run start",
+    "postinstall": "yarn build",
+    "start": "yarn build && cd packages/rn-tester && npm run start",
     "android": "cd packages/rn-tester && npm run android",
+    "build": "node ./scripts/build/build.js",
+    "clean": "node ./scripts/build/clean.js",
+    "watch": "yarn build && node ./scripts/build/watch.js",
```

Comment:
I think there's no need for an extra `watch` script.
We can just have a "single build script" which takes `--watch`.
That would be awesome.
WDYT?